cedar-policy/CHANGELOG.md

```diff
@@ -19,6 +19,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `wasm` Cargo feature for targeting Wasm
 - For the `partial-eval` experimental feature: added
 `Authorizer::evaluate_policies_partial()` (#474)
+- Exposed cedar-wasm functionality for policies and templates: `check_parse_policy_set`,
+  `policy_text_to_json`, and `policy_text_from_json`. (#616)
```

Comment:
Not true of the current code, right? Maybe you want a separate CHANGELOG in `cedar-wasm`?

Reply:
These are the current names for the `cedar-wasm` functions. What's not current anymore is templates, we aren't exposing those anymore in this PR. Is this CHANGELOG only for `cedar-policy`? I assumed it was for the whole project, but I see now there's a `cedar-policy-cli` changelog.

Reply:
The CHANGELOG here is only for the `cedar-policy` crate. We use one here and in `cedar-policy-cli` since these are the "public" parts of our project. We do not use them in other sub-folders since they contain "internal" crates that don't follow semver.